src/videojs.ima.js

```diff
@@ -220,6 +220,7 @@
       if (adsManager) {
         adsManager.destroy();
       }
+      adContainerDiv.style.display = 'none';
```

Comment:
Yea actually I'm holding off on this - I'm not able to replicate the issue as reported. I'll wait for more info and change this if needed.